Makefile

```diff
@@ -398,9 +398,8 @@ new-major-release-tag: check-branch check-tag-message
 smithyctl/cmd/bin:
 	$(eval GOOS:=linux)
```

Comment:
As mentioned in the description - dealing with SQLite means that we need C bindings